### ISSUE TITLE
fileDownloader.database.deleteRecordWithId() does not delete the file from the disk

### ISSUE THREAD

**Issue by gibahjoe** [781flyingdutchman/background_downloader#103]
**Describe the bug**
When I call ```fileDownloader.database.deleteRecordWithId(e.taskRecord!.taskId)```, the download is deleted however, the file itself is not deleted on disk.

**To Reproduce**
1. Call ```deleteRecordWithId()``` on any download

**Expected behavior**
I expect the files to be deleted on disk along with the download records else we will have a lot of orphaned files with no reference to them

**Logs**
If possible, include logs that capture the issue:
```
logs
```

**Code**
If possible, paste in the code snippet where you interact with the package:
```
Code
```

**Additional context**
If this is not the behavior the library would like to support, then it should be added the dartdoc on the delete method and even in the project documentation to let the user know that it doesn't delete the files on the disk.

**Comment by 781flyingdutchman** [comment]
Hi, the method says deleteRecordWithId - no mention of the file which indeed it doesn't delete. If you also want to delete the file, then do that before deleting the record. You can get the path to the file from the Task stored within the record.

**Comment by gibahjoe** [comment]
Yeah. That's what I am doing now. Thank you. Maybe update the documentation to remove the risk of files being downloaded into the app and then the reference being lost when you call fileDownloader.database.deleteRecordWithId thereby orphaning them.

Either way, thank you.
